WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -1,6 +1,13 @@
 <% competition = delegate_report.competition %>
 
 <div class="delegate-report">
+  <% if delegate_report.wrc_feedback_requested %>
+    <font color="red">
+      <strong>@WRC: Feedback requested on incidents: <%= delegate_report.wrc_incidents %></strong>
+    </font>
+    <br><br>
```

Comment:
Just an FYI rather than a request for change: you could wrap that in a `p` which would "naturally" introduce some bottom margin.
Also you could have considered using/extending the `alert` css class defined [here](https://github.com/thewca/worldcubeassociation.org/blob/cbcbb933a056bdf68de97e5e3a51bc93f7c3d9b3/WcaOnRails/app/assets/stylesheets/email.css.scss).

Interestingly @Jambrose777 had a similar use case of displaying red bolded messages, and created a [css class](https://github.com/thewca/worldcubeassociation.org/pull/4144/files#diff-c26f23fb9893ea57525d67d37a8cb1f0) defining just what you did through `font` and `strong` :)
(the PR is not merged yet though, so you can't just use it now. But you can keep it in mind for some upcoming work)

Reply:
Ah, good to know! thank you :)

Reply:
Did some testing (looking into doing WDC feedback), and it looks like the same style sheet doesn't apply when viewing the report within the WCA site, unfortunately :(